main.py

```diff
@@ -2,6 +2,38 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
+    genres = ["Western", "Action", "Dramma"]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    (
+        Actor.objects.filter(last_name="Reaves").update(
+            first_name="Keanu", last_name="Reeves"
+        )
+    )
```

Comment:
```suggestion
    Actor.objects.filter(last_name="Reaves").update(
        first_name="Keanu", last_name="Reeves"
    )
```
You do not need this brackets